exporter/humioexporter/config.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021, OpenTelemetry Authors
```

Comment:
Sorry for missing this in the original review, but we are now leaving the year out.

```suggestion
// Copyright The OpenTelemetry Authors
```